warehouse/README.md

```diff
@@ -38,3 +39,30 @@ docker-compose up -d
 
 Then use any db client to connect to `localhost:5431` using the db credentials
 in `.env`.
+
+## Data Organization
+
+This warehouse currently stores the cleaned and merged source data from Kids First studies which have been ingested using the ingest [library](https://github.com/kids-first/kf-lib-data-ingest) and [packages](https://github.com/kids-first/kf-ingest-packages).
+
+The cleaned source data tables are produced by ingest library's extract stage and the merged data tables are produced by the library's transform stage. Any column name in any of these tables will be a concept attribute from the library's [standard concept schema](https://github.com/kids-first/kf-lib-data-ingest/blob/master/kf_lib_data_ingest/common/concept_schema.py).
+Upon ingest of a study, a new database named after the study's KF ID will be created with two schemas: 1) `Extract` and 2) `Transform`. The below figure illustrates the organization of ingested data:
+
+[Draw.io Source](https://drive.google.com/file/d/1IJGg6Dw9TeNzLzI-9fnUvo71Y6HL8UHN/view)
+
+![Data Organization](./static/img/data-organization.png)
+
+### `Extract` Schema
+
+The `Extract` namespace stores an output(s) extracted against an extract config(s) with a table name(s) corresponding to an extract config file name(s). The following SQL statement will select all rows extracted from an extract config `participants.py`: 
+
+```sql
+SELECT * FROM "Extract"."participants";
+```
+
+### `Transform` Schema
+
+The `Transform` namespace stores an output(s) transformed by a transform module (`transform_module.py`) with a table name(s) corresponding to a Python dictionary key name(s) declared at the end of the module. The following SQL statement will select all rows from a transformed table with `DEFAULT_KEY`: 
```

Comment:
Could you please change to:

The `Transform` namespace stores transform stage output table(s) where table name(s) are set to a key name in the Python dictionary declared at the end of the transform module (`transform_module.py`) as the output of the transform stage. 

For example, if the transform module outputs the following Python dict: 
```
{
    "default": <Pandas DataFrame>,
    "family-relationship": <Pandas DataFrame>
}
```

Then, the following SQL statement will select all rows from the transform output table with the key name `default`: 
```sql
SELECT * FROM "Transform"."default";
```

Reply:
Changed!